application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java

```diff
@@ -13,7 +13,7 @@
 
             __Do not post your question in other channels.__
                  
-            After you are satisfied with the given answer don't forget to free the channel for others with the '/free' command!
+            Once your issue is resolved, feel free to /free the channel, thanks.
```

Comment:
I think we should put the `/free` into code-highlighting, like \`/free\`. Otherwise, I like it, thanks.